sdk/cosmos/azure-spring-data-cosmos/README.md

```diff
@@ -129,6 +133,7 @@ public class AppConfiguration extends AbstractCosmosConfiguration {
     public CosmosConfig cosmosConfig() {
         return CosmosConfig.builder()
                            .enableQueryMetrics(queryMetricsEnabled)
+                           .setMaxDegreeOfParallelism(maxDegreeOfParallelism)
```

Comment:
Once you update the API name, please update all the samples as well, thanks!

Reply:
Done.

Reply:
@trande4884 - may be we just need to update the samples that you have created with the correct API name.